processing_handler.go

```diff
@@ -251,6 +252,11 @@ func handleProcessing(reqID string, rw http.ResponseWriter, r *http.Request) {
 	metrics.SetMetadata(ctx, "imgproxy.source_image_url", imageURL)
 	metrics.SetMetadata(ctx, "imgproxy.processing_options", po)
 
+	parsedURL, err := url.Parse(imageURL)
+	if err == nil {
```

Comment:
Please use short notation
```suggestion
	if u, err := url.Parse(imageURL); err != nil {
```

Reply:
Thanks.
I updated with the following line.